src/js/utils/ValidatorUtil.js

```diff
@@ -1,10 +1,33 @@
 var ValidatorUtil = {
+  isDefined: function (value) {
+    return value != null && value !== '' || typeof value === 'number';
+  },
+
   isEmail: function (email) {
     return email != null &&
       email.length > 0 &&
       !/\s/.test(email) &&
       /.+@.+\..+/
       .test(email);
+  },
+
+  isInteger: function (value) {
+    const number = parseFloat(value);
+
+    return Number.isInteger(number);
+  },
+
+  isNumber: function (value) {
+    const number = parseFloat(value);
+
+    return !Number.isNaN(number) && Number.isFinite(number);
+  },
+
+  isNumberInRange: function (value, range = {}) {
```

Comment:
Minor comment, but I would also add an `exclusive (boolean)` parameter. When `true` do only `>` or `<` tests.
For example: `isNumberInRange( numberOfCPUs, 0, 65536, true );`, since a user can enter a 0.01 as value.